conf/.bashrc

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 export PGHOST=/tmp
-export PATH=/usr/pgsql-${PGVERSION}/bin:/opt/cpm/bin:$PATH
+export PATH=/usr/pgsql-${PGVERSION}/bin:/opt/crunchy/bin:$PATH
 export LD_LIBRARY_PATH=/usr/pgsql-${PGVERSION}/lib
 
 # export LD_PRELOAD=/usr/lib64/libnss_wrapper.so
```

Comment:
Remove this.

Reply:
this export is also include here: https://github.com/CrunchyData/crunchy-containers/blob/f7236965d8d022306ebddd9b8c15e42bf1aabdd6/conf/.bash_profile#L4

Should I remove that too?

Reply:
honestly do we even need both of these files?

Reply:
removed both bash files and went through some basic testing using docker and ocp 3.11